src/transforms/glossary.ts

```diff
@@ -50,11 +50,11 @@ export function addGlossaryDefinition(
         let [glossary, glossaryPath] = Editor.above(editor, {
             at: path,
             match: Glossary.isGlossary,
-        })!
+        }) ?? []
 
         let index
 
-        if (glossary == null) {
+        if (glossary == null || glossaryPath == null) {
```

Comment:
This is not necessary. `Editor.above` returns `[Element, Path]`, so if `glossary == null` then also `glossaryPath == null`.

Reply:
then i'll have to use non-null assertion in `glossaryPath!`